solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/issue_pass.rs

```diff
@@ -22,6 +22,7 @@ pub fn issue_pass(ctx: Context<IssuePass>, subject: Pubkey, pass_number: u16) ->
     let gatekeeper_ata = &mut ctx.accounts.gatekeeper_token_account;
     let funder_ata = &mut ctx.accounts.funder_token_account;
 
+    // TODO: Can we put the fee transfers into a trait and reuse dependent on the type of instruction?
```

Comment:
@JulianSpring and I looked into it... the problem is `.issue` at the end... rust doesn't have any means to access variables dynamically (at least, not in any iffiencient way from what I can tell)